internal/search/query/transformer.go

```diff
@@ -109,3 +109,11 @@ func containsUppercase(s string) bool {
 	}
 	return false
 }
+
+// Map pipes query through one or more query transformer functions.
+func Map(query []Node, fns ...func(_ []Node) []Node) []Node {
```

Comment:
```suggestion
func Map(query []Node, fns ...func([]Node) []Node) []Node {
```

No need for underscore.